mne/datasets/config.py

```diff
@@ -95,123 +95,270 @@
-)
-FOLDER_NAMES = dict(
-    bst_auditory='MNE-brainstorm-data',
-    bst_phantom_ctf='MNE-brainstorm-data',
-    bst_phantom_elekta='MNE-brainstorm-data',
-    bst_raw='MNE-brainstorm-data',
-    bst_resting='MNE-brainstorm-data',
-    erp_core='MNE-ERP-CORE-data',
-    epilepsy_ecog='MNE-epilepsy-ecog-data',
-    fake='foo',
-    fieldtrip_cmc='MNE-fieldtrip_cmc-data',
-    fnirs_motor='MNE-fNIRS-motor-data',
-    hf_sef_raw='HF_SEF',
-    hf_sef_evoked='HF_SEF',
-    kiloword='MNE-kiloword-data',
-    misc='MNE-misc-data',
-    mtrf='mTRF_1.5',
-    multimodal='MNE-multimodal-data',
-    opm='MNE-OPM-data',
-    phantom_4dbti='MNE-phantom-4DBTi',
-    refmeg_noise='MNE-refmeg-noise-data',
-    sample='MNE-sample-data',
-    somato='MNE-somato-data',
-    spm='MNE-spm-face',
-    ssvep='ssvep-example-data',
-    testing='MNE-testing-data',
-    visual_92_categories='MNE-visual_92_categories-data',
-)
-
-# the environment configuration keys for each dataset name
-CONFIG_KEYS = dict(
-    bst_auditory='MNE_DATASETS_BRAINSTORM_PATH',
-    bst_phantom_ctf='MNE_DATASETS_BRAINSTORM_PATH',
-    bst_phantom_elekta='MNE_DATASETS_BRAINSTORM_PATH',
-    bst_raw='MNE_DATASETS_BRAINSTORM_PATH',
-    bst_resting='MNE_DATASETS_BRAINSTORM_PATH',
-    erp_core='MNE_DATASETS_ERP_CORE_PATH',
-    epilepsy_ecog='MNE_DATASETS_EPILEPSY_ECOG_PATH',
-    fake='MNE_DATASETS_FAKE_PATH',
-    fieldtrip_cmc='MNE_DATASETS_FIELDTRIP_CMC_PATH',
-    fnirs_motor='MNE_DATASETS_FNIRS_MOTOR_PATH',
-    hf_sef_raw='MNE_DATASETS_HF_SEF_PATH',
-    hf_sef_evoked='MNE_DATASETS_HF_SEF_PATH',
-    kiloword='MNE_DATASETS_KILOWORD_PATH',
-    misc='MNE_DATASETS_MISC_PATH',
-    multimodal='MNE_DATASETS_MULTIMODAL_PATH',
-    mtrf='MNE_DATASETS_MTRF_PATH',
-    opm='MNE_DATASETS_OPM_PATH',
-    refmeg_noise='MNE_DATASETS_REFMEG_NOISE_PATH',
-    phantom_4dbti='MNE_DATASETS_PHANTOM_4DBTI_PATH',
-    sample='MNE_DATASETS_SAMPLE_PATH',
-    spm='MNE_DATASETS_SPM_FACE_PATH',
-    somato='MNE_DATASETS_SOMATO_PATH',
-    ssvep='MNE_DATASETS_SSVEP_PATH',
-    testing='MNE_DATASETS_TESTING_PATH',
-    visual_92_categories='MNE_DATASETS_VISUAL_92_CATEGORIES_PATH',
-)
-assert set(ARCHIVE_NAMES) == set(URLS)
+# Each dataset should contain minimally:
+# - archive_name : the name of the compressed file that is downloaded
+# - hash : the cryptographic hash type of the file followed by a colon and then
+#          the hash value (examples: "sha256:19uheid...", "md5:upodh2io...")
+# - url : URL from which the file can be downloaded
+# - folder_name : the subfolder within the MNE data folder in which to save and
+#                 uncompress (if needed) the file(s)
+# - config_key : key to use with `mne.set_config` to store the on-disk location
+#                of the downloaded dataset (ex: "MNE_DATASETS_EEGBCI_PATH").
+# - (optional) token : An authorization token, if one is needed to access the
+#                      download URL.
+fnirs_motor = dict(
```

Comment:
@adam2392 sorry I was slow to review here. I would suggest you write it as:

```
MNE_DATASETS = dict()

MNE_DATASETS['fnirs_motor'] = dict(
```

this way you don't need to replicate below the list of datasets so you won't forget one.

can you open a new PR?